compiler/crates/relay-compiler/relay-compiler-config-schema.json

```diff
@@ -1468,6 +1468,48 @@
           "default": false,
           "type": "boolean"
         },
+        "resolverContextType": {
```

Comment:
If you rerun the Rust tests with the flag to update snapshots this should update to include the description